save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/ContestController.kt

```diff
@@ -190,7 +194,7 @@ internal class ContestController(
     ): Mono<TestFilesContent> = getContestOrNotFound(contestName)
         .zipWith(Mono.just(testSuiteId))
         .filter { (contest, testSuiteId) ->
-            testSuiteId in contest.getTestSuiteIds()
+            testSuiteId in lnkContestTestSuiteService.getAllTestSuiteIdsByContest(contest)
```

Comment:
Is it necessary to have so many checks?
Why not just do:
`contest.testSuiteLinks.map { it.testSuite }.filter { testSuiteId in it.id }` ?

Reply:
A bit reworked this endpoint